<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-MBJVQGU" failures="3" tests="4" name="BankingApp Test(failed)" time="251.955" errors="0" timestamp="2022-08-02T16:49:09 IST">
  <testcase classname="com.inetBanking.testCase.TC_LoginPageDDT_002" name="loginDDT" time="12.377">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCase.TC_LoginPageDDT_002.loginDDT(TC_LoginPageDDT_002.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetBanking.testCase.TC_LoginPageDDT_002" name="loginDDT" time="8.176">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCase.TC_LoginPageDDT_002.loginDDT(TC_LoginPageDDT_002.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetBanking.testCase.TC_LoginPageDDT_002" name="loginDDT" time="4.462">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCase.TC_LoginPageDDT_002.loginDDT(TC_LoginPageDDT_002.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetBanking.testCase.TC_LoginPage_001" name="loginTest" time="2.948"/>
</testsuite> <!-- BankingApp Test(failed) -->
